homeassistant/components/google_assistant_sdk/config_flow.py

```diff
@@ -108,6 +114,12 @@ async def async_step_init(
                         CONF_LANGUAGE_CODE,
                         default=self.config_entry.options.get(CONF_LANGUAGE_CODE),
                     ): vol.In(SUPPORTED_LANGUAGE_CODES),
+                    vol.Required(
+                        CONF_ENABLE_CONVERSATION_AGENT,
```

Comment:
If you don't see the string, refresh your browser. You did commit `translations/en.json` so that should show.

Reply:
Works now thanks.